packages/ui/src/components/SideMenu/SideMenuItem.tsx

```diff
@@ -89,12 +89,12 @@ const SideMenuItem = ({ icon, children, href, ...rest }: SideMenuItemProps) => {
     if (isSmallScreen && ctx?.onOpenChange) {
       ctx?.onOpenChange(false);
     }
-    navigate(url || "");
+    navigate(url ?? "");
```

Comment:
Pre-existing: Why does this one coalesce to "" but L97 to "#".  Why would we pass an empty string to `navigate` anyway?